pkg/platform/common/vmlayer/vmparams.go

```diff
@@ -587,6 +587,9 @@ type VMOrchestrationParams struct {
 	AuthPublicKey           string
 	DeploymentManifest      string
 	Command                 string
+	// TODO - Volumes should be just a reference here and
+	// and the volume definition should be in VMGroupOrchestrationParams
+	// similar how ports and handled.
```

Comment:
Tracked in [this issue](https://github.com/edgexr/edge-cloud-platform/issues/335)